src/database/sql_database.h

```diff
@@ -188,6 +189,9 @@ class SQLDatabase : public Database {
     /// \brief migrate resources from mt_cds_objects to grb_resource before removing the column (DBVERSION 13)
     bool doResourceMigration();
     void migrateResources(int objectId, const std::string& resourcesStr);
+    
+    /// \brief returns a fmt-prinable identifier name
+    SQLIdentifier identifier(const std::string_view& name) const { return SQLIdentifier(name, table_quote_begin); }
```

Comment:
there's no need to take a string_view by reference. It's cheap to copy.

Reply:
This is already fixed in a previous commit